mne/viz/tests/test_evoked.py

```diff
@@ -354,5 +355,31 @@ def test_plot_ctf():
     evoked.plot_joint(times=[0.1])
     mne.viz.plot_compare_evokeds([evoked, evoked])
 
+    # make sure axes position is "almost" unchanged
+    # when axes were passed to plot_joint by the user
+    times = [0.1, 0.2, 0.3]
+    fig = plt.figure()
+
+    # create custom axes for topomaps, colorbar and the timeseries
+    gs = gridspec.GridSpec(3, 7, hspace=0.5, top=0.8)
+    topo_axes = [fig.add_subplot(gs[0, idx * 2:(idx + 1) * 2])
+                 for idx in range(len(times))]
+    topo_axes.append(fig.add_subplot(gs[0, -1]))
+    ts_axis = fig.add_subplot(gs[1:, 1:-1])
+
+    def get_axes_midpoints(axes):
```

Comment:
if someone knows a better way to get axis midpoint I'd be happy to change that. I could use axis transform and figure inverse transform to get midpoint position, but I'm not sure it would be better.